composer.json

```diff
@@ -27,12 +27,20 @@
         "mksanitizedparameters": "self.version",
         "typo3-ter/mksanitizedparameters": "self.version"
     },
+    "repositories": [{
+        "type": "vcs",
+        "url": "https://github.com/Baachi/phpstan-typo3.git"
+    }
+    ],
     "require": {
         "typo3/cms-core": "~8.7 || ~9.5",
         "digedag/rn-base": ">=1.10.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2@stable"
+        "friendsofphp/php-cs-fixer": "~2@stable",
+        "phpstan/phpstan": "^0.11",
+        "saschaegerer/phpstan-typo3": "dev-feature/update-phpstan",
+        "typo3/testing-framework": "*"
```

Comment:
Please use a specific version and not the "star of death" :smile:

Reply:
You're right. I've tried to find a suitable version of typo3/testing-framework, but version 2.0.4, which support typo3/cms-core:"^8.5 || ^9", not support phpunit 7. I think, downgrade phpunit isn't the way to go.